.github/ISSUE_TEMPLATE/release_testnet.md

```diff
@@ -15,39 +15,40 @@ Testnet release manager: X
 
 # Testnet Release Manager Checklist
 
-Monday (week before release):
+Preceding Friday (sprint planning day):
 
 - [ ] Create GitHub project column, work with team to populate the milestone with tickets targeted for the release.
 
+Tuesday (or after release of previous testnet):
+
+- [ ] Construct the genesis data for the release:
+  - [ ] Update `discord_history.csv` file using the Galileo bot (`cd galileo && cargo run --release history --channel https://discord.com/channels/824484045370818580/915710851917439060 > ../penumbra/testnets/discord_history.csv`, assuming that the galileo and penumbra repos are sitting side-by-side in the file system)
```

Comment:
Can we automate this with a Github Action that runs this command and submits a PR with the changes to `discord_history.csv`?

Reply:
I would love to! I don't think there's an adequate action we can trigger on; we don't want to trigger based on a tag push to `penumbra-zone/penumbra`, because we want the allocations in the latest tag. Zooming out a bit, moving the galilelo history task to Tuesday (i.e. post-release) means that the subsequent testnet will have stale data: any allocations requested in Discord over the past week would be left out.

Which suggests we should just use a time-based schedule to trigger the job: on Monday mornings, a Galileo job should fire automatically, pull the latest info from the Discord API, and submit a PR to `penumbra-zone/penumbra`. That would be super helpful.

I'd prefer to tackle that as part of automating the galileo updates for testnet changes. Absent that automation, I feel it may be best to keep the pre-diff schedule of "Monday (release day)". Since following this checklist is a bit tedious, I hereby volunteer to do the tasks manually, which will absolutely motivate me to automate them. :grinning:

Reply:
Sounds great, updating the discord addresses should be done on a best-effort basis, it's not worth disrupting the workflow over.